src/styles/cards.scss

```diff
@@ -0,0 +1,136 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+
+    &--y {
+      margin: 40px 0 4px;
+    }
+  }
+
+  &:hover &__text {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $minor-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    &--y {
+      margin-bottom: 16px;
+    }
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $minor-text-color;
+    }
+
+    &-text {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--y {
+    margin: 16px 0 24px;
+  }
+  &__wrap {
+    display: flex;
+  }
+
+  &__star {
+    display: block;
+    width: 16px;
+    height: 16px;
+    background: url(./../images/star.svg) no-repeat center center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n+4) {
+      background-image: url(./../images/star-active.svg);
+    }
```

Comment:
Implement this part from checklist 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/be4a78db-62aa-41c3-9a5b-2fd01ab2dae1)